conn.go

```diff
@@ -661,6 +679,7 @@ func (c *Conn) Subscribe(destination string, ack AckMode, opts ...func(*frame.Fr
 	closeMutex := &sync.Mutex{}
 	sub := &Subscription{
 		id:          id,
+		replyToSet:  replyToSet,
```

Comment:
nit: thoughts about renaming this to `shouldReplyTo`?

Reply:
Hi, because in normal case id is used as map key in function processLoop, I think in replyToSet case id should be set the same way to keep consistent, then if id is set to value of ReplyToHeader, and the added field is set to this too, this will be duplication, so I name it that way. You think it's better to rename to shouldReplyTo?